rollrus.go

```diff
@@ -9,20 +9,42 @@ import (
 	"github.com/stvp/roll"
 )
 
+var defaultFiredLevels = []log.Level{
+	log.ErrorLevel,
+	log.FatalLevel,
+	log.PanicLevel,
+}
+
 // Hook wrapper for the rollbar Client
 // May be used as a rollbar client itself
 type Hook struct {
 	roll.Client
+	firedLevels []log.Level
 }
 
 // SetupLogging sets up logging. if token is not and empty string a rollbar
 // hook is added with the environment set to env. The log formatter is set to a
 // TextFormatter with timestamps disabled, which is suitable for use on Heroku.
 func SetupLogging(token, env string) {
+	setupLogging(token, env, defaultFiredLevels)
+}
+
+// SetupLoggingForLevels sets up logging. if token is not and empty
+// string a rollbar hook is added with the environment set to env. The
+// log formatter is set to a TextFormatter with timestamps disabled,
+// which is suitable for use on Heroku.
```

Comment:
Possibly just say:

// SetupLoggingForLevels works like SetupLogging, but allows you to set the levels on which to trigger this hook.